test/e2e-kuttl/quota-forest/10-install.yaml

```diff
@@ -0,0 +1,61 @@
+---
+apiVersion: mcad.ibm.com/v1beta1
+kind: AppWrapper
+metadata:
+    name: job-without-quota
+    namespace: test
```

Comment:
Please add a labels section, to ensure that any other user supplied labels are preserved.

Reply:
Done